pkgs/development/python-modules/onnx/default.nix

```diff
@@ -39,10 +40,14 @@ in buildPythonPackage rec {
 
   buildInputs = [
     abseil-cpp
+    pythonPackages.protobuf
+    pythonPackages.google-re2
+    pythonPackages.pillow
   ];
 
   propagatedBuildInputs = [
-    protobuf
+    protobuf_21
+    pythonPackages.protobuf
```

Comment:
I don't think we need to propdate both. Also is the python package not already including the cli?

Reply:
The build fails using only the Python package with an absl errors like the following:
```
In file included from /tmp/nix-build-python3.11-onnx-1.15.0.drv-1/source/.setuptools-cmake-build/onnx/onnx-operators-ml.pb.cc:4:
In file included from /tmp/nix-build-python3.11-onnx-1.15.0.drv-1/source/.setuptools-cmake-build/onnx/onnx-operators-ml.pb.h:24:
In file included from /nix/store/glnr34k9z7qwrhcpnrp77b83h3mgfhiq-protobuf-24.4/include/google/protobuf/io/coded_stream.h:134:
In file included from /nix/store/mid900708j8vgfrkd5yfxn48f7x4pspc-abseil-cpp-20230125.3/include/absl/strings/cord.h:78:
In file included from /nix/store/mid900708j8vgfrkd5yfxn48f7x4pspc-abseil-cpp-20230125.3/include/absl/container/inlined_vector.h:53:
In file included from /nix/store/mid900708j8vgfrkd5yfxn48f7x4pspc-abseil-cpp-20230125.3/include/absl/container/internal/inlined_vector.h:30:
In file included from /nix/store/mid900708j8vgfrkd5yfxn48f7x4pspc-abseil-cpp-20230125.3/include/absl/container/internal/compressed_tuple.h:40:
/nix/store/mid900708j8vgfrkd5yfxn48f7x4pspc-abseil-cpp-20230125.3/include/absl/utility/utility.h:164:12: error: no member named 'in_place_t' in namespace 'std'
```

Using only `protobuf_21` won't work since protobuf is use from Python in the final package.